sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/test/public/util/setup.browser.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from "@opentelemetry/sdk-trace-base";
+
+import { OpenTelemetryInstrumenter } from "../../../src/instrumenter";
+import { useInstrumenter } from "@azure/core-tracing";
+
+// Setup all the necessary instrumenters, exporters, etc.
+export const inMemoryExporter = new InMemorySpanExporter();
+// Use BasicTracerProvider in the browser.
+const provider = new BasicTracerProvider();
```

Comment:
this is the only difference, right? Maybe we can put returning the provider into browser/node specific files and leave the rest of this the same for ease of maintenance?

Reply:
Thanks, fixed!